src/docs/src/translation/ms.json

```diff
@@ -0,0 +1,469 @@
+  "daisyUI adds some style to @tailwindcss/typography so it will use the same theme as other elements.": "daisyUI menambah beberapa gaya pada @tailwindcss/typography supaya ia akan menggunakan tema yang sama seperti elemen lain.",
+  "Make sure you require <span class='badge badge-outline'>daisyui</span> AFTER <span class='badge badge-outline'>@tailwindcss/typography</span> in tailwind.config.js": "Pastikan anda memerlukan <span class='badge-outline'>daisyui</span> SELEPAS <span class='badge-outline'>@tailwindcss/typography</span> dalam tailwind.config.js",
+  "Here you can see how elements will look using <code>@tailwindcss/typography</code>.": "Di sini anda boleh melihat rupa elemen menggunakan <code>@tailwindcss/typography</code>.",
+  "To use a custom prefix, <a class='link' href='/docs/config/#prefix'>add your prefix string to config</a>": "Untuk menggunakan awalan tersuai, <a class='link' href='/docs/config/#prefix'>tambah rentetan awalan anda pada konfigurasi</a>",
+  "Component": "Komponen",
+  "Modifier": "Pengubah suai",
+  "Responsive": "Responsif",
+  "Changes the style of a component": "Mengubah gaya komponen",
+  "Supports responsive prefixes (sm:, lg:, …)": "Menyokong awalan responsif (sm:, lg:, …)",
+  "Make your theme": "Buat tema anda",
+  "More examples": "Lagi contoh",
+  "Support daisyUI's development": "Sokong pembangunan daisyUI",
+  "The most popular": "Yang paling popular",
+  "component library": "perpustakaan komponen",
+  "for Tailwind CSS": "untuk Tailwind CSS",
+  "daisyUI adds component class names to Tailwind&nbsp;CSS<br /> so you can make beautiful websites <span class='border-b-2 border-base-content/20'>faster than ever.</span>": "daisyUI menambah nama kelas komponen pada Tailwind&nbsp;CSS<br /> supaya anda boleh membuat laman web yang cantik <span class='border-b-2 border-base-content/20'> dan lebih pantas berbanding sebelum ini.</span>",
+  "don't re-invent <br/>the wheel <br/>every time": "jangan cipta semula <br/>roda <br/>setiap kali",
+  "In a Tailwind CSS project, you need to write utility class names for every element. Thousands of class names just to style the most basic elements.": "Dalam projek Tailwind CSS, anda perlu menulis nama kelas utiliti untuk setiap elemen. Beribu-ribu nama kelas hanya untuk menggayakan elemen paling asas.",
+  "instead of writing<br /> <span class='text-error'><span class='font-black'>100</span> class names</span>": "daripada menulis<br /> <span class='text-error'><span class='font-black'>100</span> nama kelas</span>",
+  "For every element, every page, every project,<br/>again and again": "Untuk setiap elemen, setiap halaman, setiap projek,<br/>berulang kali",
+  "use <span class='text-success'><span class='font-black'>semantic</span> <br />class names</span>": "gunakan <span class='text-success'><span class='font-black'>'Semantic'</span> <br />nama kelas</span>",
```

Comment:
Does `'Semantic'` need that quotation marks?

Reply:
Yes, cause im using both malay and english for that sentence. In order to differentiate those two, i included that quotation marks.